src/main/java/uk/gov/justice/probation/courtcaseservice/application/ApplicationExceptionHandler.java

```diff
@@ -26,6 +27,14 @@
                 .userMessage(e.getMessage()).build(), NOT_FOUND);
     }
 
+    @ExceptionHandler(DocumentNotFoundException.class)
```

Comment:
I've implemented a generic `RestResourceNotFound` exception and handler as part of my changes as it happens: https://github.com/ministryofjustice/court-case-service/pull/79

If mine gets merged first could consider pulling from master and applying the same pattern and could get rid of this handler

Reply:
Cheers - think it's going to be the other way around, will help merge